### ISSUE TITLE
Checkboxes are not reacting properly to changes

### ISSUE THREAD

**Issue by dominicbarnes** [dekujs/todomvc#1]
In this app if I remove a checked item from the list, the checkbox for that row and the one below get into a bad state. (hard to explain precisely, better to try it out for yourself)

This is most likely a pooling issue, but since I don't have globally unique IDs for these todo items, I can't rely on using a `key`. Besides, a simple example like this shouldn't require a more advanced feature like that imo.

**Comment by dominicbarnes** [comment]
Here's a gif of the bug in action:

![](http://g.recordit.co/wdtSiS6vW7.gif)

**Comment by anthonyshort** [comment]
I'll have to take a deeper look but I'm thinking it's either pooling or we're not handling the `checked` attribute properly

**Comment by dominicbarnes** [comment]
Yeah, looks like we need to add special-cases for attributes like `checked` and `disabled`.

Deku is adding `checked="true"` as an HTML attribute, and the raw element shows `el.checked === false`. Looks like we need to take those cases and set both the DOM property _and_ the HTML attribute.
